docs/install-understack-ubuntu-k3s.md

```diff
@@ -0,0 +1,305 @@
+# Installing UnderStack on Ubuntu 22.04 + K3s
+
+## Install Pre-requisites
+
+Install some packages we'll need later and some useful troubleshooting utilities.
```

Comment:
are we installing full golang just to have the `go env GOOS` and `go env GOARCH` available?
If so, I think we can safely assume that Ubuntu is `linux` and we can get architecture with `uname -m` although I don't think we will be running ARM servers that soon.

Reply:
Yeah, I was following their docs which used it and I wasn't sure if I would need it for anything else. I've updated it to remove the go dependency. 👍